lambdas/get-addresses/jest.config.ts

```diff
@@ -5,11 +5,11 @@
 
 export default {
     transform: {
-        "^.+\\.ts?$": "esbuild-jest",
+        "^.+\\.ts?$": "ts-jest",
```

Comment:
use ts-jest to build tests and not esbuild.

esbuild + breakpoints on vscode was acting strange and not breakpointing in the right locations. The log lines were also slightly incorrect for the best output, it was strange.